### PR TITLE
Bug 1097754 - test_dialer_receive_call_with_locked_screen.py sometimes d...

### PR DESCRIPTION
**Pull request by JohanLorenzo**
...oesn't wait until the callscreen arrives at the bottom of the screen

### CODE REVIEW

File: tests/python/gaia-ui-tests/gaiatest/apps/phone/regions/call_screen.py [hunk 1]
```diff
@@ -12,6 +14,9 @@ class CallScreen(Phone):
     MAX_NUMBER_OF_DISPLAYED_DIGITS = 22
 
     _call_screen_locator = (By.CSS_SELECTOR, "iframe[name='call_screen']")
+    # See https://github.com/mozilla-b2g/gaia/blob/master/apps/callscreen/index.html#L79
+    # FIXME: Find an explanation for the name of the following ID
```
Comment (JohanLorenzo) [inline tests/python/gaia-ui-tests/gaiatest/apps/phone/regions/call_screen.py]: I'm investigating this with the Dialer team.

**Comment by try-server-hook** [comment]
[JohanLorenzo Johan Lorenzo (JohanLorenzo)](https://github.com/JohanLorenzo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=0ca3d80d737d)

**Comment by try-server-hook** [comment]
[JohanLorenzo Johan Lorenzo (JohanLorenzo)](https://github.com/JohanLorenzo) started tests. [Results](https://treeherder.mozilla.org/ui/#/jobs?repo=gaia-try&revision=c943c5cac38e)

**Comment by viorelaioia** [comment]
Lgtm, merging

**Pull request merged**